### ISSUE TITLE
Hydlide 2 Patched doesn't work correctly

### ISSUE THREAD

**Issue by TFHFony** [ppeccin/WebMSX#20]
When issuing the English Translation patch to the Hydlide ROM it doesn't work anymore on WebMSX.
IPS Patch: http://ips.tni.nl/rom/translations/
Patched file: http://www.file-hunter.com/MSX/Hydlide2.zip
This version works, but you can't save the player you have created (see this issue: http://www.romhacking.net/forum/index.php?topic=14328.0
After manually selecting the ASCI16SRAM2 mapper, the game crashes:
Patched file with correct rom-type: http://www.file-hunter.com/MSX/Hydlide2-2.zip
It seems that RuMSX even made a special ROM type for Hydlide II: http://www.emucr.com/2017/06/rumsx-v082.html

**Comment by ppeccin** [comment]
I see...
Do you know which mapper it uses?
Is it really ASCII16SRAM2 or is it a special mapper?

If its a special mapper, we need to find the details about it so I can add support.
Does the original, not patched ROM work with ASCI16SRAM2  mapper? 

Paulo

**Comment by TFHFony** [comment]
Hi Paulo!

The original I test with, I downloaded here:
http://www.planetemu.net/rom/goodmsx1/hydlide-2-1986-t-and-e-soft-j
This original file is recognized as ASCII16SRAM2 by WebMSX. It does boot,
but it doesn't work as it should. The colors when creating a character in
the main menu aren't correct and when you actually want to start playing
the game, you get a menu instead of the game.
I have tested the original file & the translated ones in BlueMSX. If I
manually configure BluMSX as ASCII16SRAM, the files do work fine.

Kind Regards,

Arnaud
